### PR TITLE
Simple implementation of a Open/Close container event for 1.10

### PR DESCRIPTION
**Pull request by luacs1998**
As mentioned on IRC.

I'm not sure how clean-room this is, but it should be a good enough replacement for the PlayerOpenContainerEvent that had to be removed with the relicense.

Comments? Feedback?

(BTW, I've signed the CLA.)

### CODE REVIEW

File: src/main/java/net/minecraftforge/event/entity/player/PlayerContainerEvent.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.event.entity.player;
```
Comment (kashike) [inline src/main/java/net/minecraftforge/event/entity/player/PlayerContainerEvent.java]: I assume you should be including the license header here.

File: src/main/java/net/minecraftforge/event/entity/player/PlayerContainerEvent.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import net.minecraft.inventory.Container;
+
+public class PlayerContainerEvent extends PlayerEvent
+{
+    public final Container container;
+    public PlayerContainerEvent(EntityPlayer player, Container container)
+    {
+        super(player);
+        this.container = container;
+
```
Comment (kashike) [inline src/main/java/net/minecraftforge/event/entity/player/PlayerContainerEvent.java]: Unnecessary empty line

**Comment by diesieben07** [comment]
Much more useful than the older event, +1.

**Comment by williewillus** [comment]
the convention for events nowadays is private final field and public getter I believe

**Comment by cpw** [comment]
Looks good to me. Please follow the new event convention (getter and private final field)

File: src/main/java/net/minecraftforge/event/entity/player/PlayerContainerEvent.java (hunk 1 above)
Comment (luacs1998) [inline src/main/java/net/minecraftforge/event/entity/player/PlayerContainerEvent.java]: How would I do that? Do I run anything in gradle?

**Comment by luacs1998** [comment]
Fixed convention.

**Pull request merged**